### DISCUSSION TITLE
The PDF preview returns blank for the updated version

### DISCUSSION THREAD

**Discussion by songhanzhang** [James-Yu/LaTeX-Workshop#3148] [Q&A]
Today I find that the PDF preview does open a new tab, but the page shows entirely blank.

I tried to open the PDF from external PDF viewers and webpage, and the results are all fine.

Finally, I changed to install an old version 8.18.0, and deleted the 8.19.2 version, the issue has been solved!

I think there might be some bug in the latest version.

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.

The issue might be a duplicate of #2743. Please start VS Code with the option `--disable-gpu`. See https://code.visualstudio.com/docs/supporting/FAQ#_vs-code-is-blank
